src/Microsoft.AspNetCore.Mvc.Core/RedirectResult.cs

```diff
@@ -35,11 +40,14 @@ public RedirectResult(string url, bool permanent)
             }
 
             Permanent = permanent;
+            PreserveMethod = preserveMethod;
             Url = url;
         }
 
         public bool Permanent { get; set; }
 
+        public bool PreserveMethod { get; set; }
```

Comment:
Are we wiring this up with the corresponding Executor as part of this change? Or is it going to be done separately?